docs/doc_v1.0/Build_documentation.md

```diff
@@ -103,10 +122,16 @@ Gender balance: this is the ratio of men:women, so there are 99 men for
 every 100 women in this example. This data has been taken from Index
 Mundi.
 
-![](image4.png)
+<img src="image4.png" alt="exposureProb" width="300">
 
 Figure 4. The population input
 
+### Initial Exposures
+
+The initial exposures file contains a list of IDs of the people who are in the
+exposed category at t=0. If this list is longer than the initial exposures, it 
+will be truncated. If it is too short, it will be filled randomly. 
```

Comment:
```suggestion
The initial exposures file contains a list of IDs of the individuals who are
initially exposed at the start of the simulation, i.e. at the first time step. If this list is longer than the initial exposures, it will be truncated. If it is too short, it will be filled randomly. 
```